packages/backend/src/open_payments/payment/outgoing/worker.ts

```diff
@@ -79,10 +91,15 @@ async function handlePaymentLifecycle(
     return
   }
 
+  const stopTimer = deps.telemetry.startTimer('handle_sending_ms', {
+    callName: 'handleSending'
```

Comment:
what do you think about prefixing `callName` with service name always?

Reply:
messaged you on this with more details. seems like a good idea but if we want to make it a standard prefix I think maybe it needs to be more abstract than service name since not everything is called from outside the service method or is necessarily scoped to a service (gql resolvers,accountService name doesnt properly signify tb/psql, etc.)

Reply:
updated to prefix with a pattern like `[AbstractScopeName]:literalFunctionName`. 

for example: 
- `WebhookService:getResourceById`
- `Resolver:depositOutgoingPaymentLiquidity` 

Not every case is as clean as service calls where we have an obvious and unambiguous scope so I'm keeping it a little more abstract to ensure we can standardize.